docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -546,6 +547,16 @@ To use custom values in an Ingress rule define these annotation:
 nginx.ingress.kubernetes.io/proxy-buffering: "on"
 ```
 
+### Proxy buffer Number
```

Comment:
Please call it `buffers number` and reflect that in variables as well, i.e `Proxy.BuffersNumber` instead of `BufferNumber`.